pkg/apis/k0s/v1beta1/extensions.go

```diff
@@ -28,9 +28,22 @@ var _ Validateable = (*ClusterExtensions)(nil)
 
 // ClusterExtensions specifies cluster extensions
 type ClusterExtensions struct {
-	//+kubebuilder:deprecatedversion:warning="storage is deprecated and will be ignored in 1.30. https://docs.k0sproject.io/stable/examples/openebs".
-	Storage *StorageExtension `json:"storage"`
-	Helm    *HelmExtensions   `json:"helm"`
+	// Deprecated: storage is deprecated and will be ignored starting from k0s
+	// 1.31 and onwards: https://docs.k0sproject.io/stable/examples/openebs
```

Comment:
In 1.30 we already say it's being ignored, now we're saying we'll ignore it 1.31 onwards. bit confusing

Reply:
We said "and _will be_ ignored in 1.30" (emphasis mine), but that never happened. This PR is now fulfilling that promise.